performance/streaming-optimization.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Streaming Optimizations"
```

Comment:
We mostly follow Google Developer Documentation Style Guide and use sentence-style headings. 
https://www.notion.so/risingwave-labs/Style-Guide-RisingWave-Labs-4ced028eb5e9476ea692795a042e441d?pvs=4

Reply:
https://github.com/risingwavelabs/risingwave-docs/pull/401/commits/580bd3d437a71fa6eb264bb8d03a6ef35508b752